crates/wallet/tests/wallet.rs

```diff
@@ -222,7 +222,7 @@ fn wallet_load_checks() -> anyhow::Result<()> {
         );
         assert_matches!(
             Wallet::load()
-                .network(Network::Bitcoin)
+                .validate_network(Network::Bitcoin)
```

Comment:
Unrelated to this PR but I noticed we're testing `validate_network` twice with two wrong networks - I think once should be fine. Second, the error is `InvalidChangeSet` but it's not the changeset that is invalid, rather the params.

Reply:
Good eye. I removed the second test.

Reply:
As for the `InvalidChangeset`, I agree it's not quite right. Will ask about this in the main PR thread: it might warrant a separate commit to add a new variant on the enum.

Reply:
The way I read this error is it's saying the changeset is invalid because it doesn't match the passed in params, which I think is what we want. We're assuming the caller provides the correct values to check the changeset against.